File: benchmarks/primes.js
function isPrime(n) {
    if (n < 2) return false;
    for (let j = 2; j * j <= n; j++) {
        if (n % j === 0) return false;
    }
    return true;
}

let count = 0;
for (let i = 2; i <= 100000; i++) {
    if (isPrime(i)) count++;
}
process.exit(count % 256);

File: benchmarks/primes.lua
local function is_prime(n)
    if n < 2 then return false end
    local j = 2
    while j * j <= n do
        if n % j == 0 then return false end
        j = j + 1
    end
    return true
end

local count = 0
for i = 2, 100000 do
    if is_prime(i) then count = count + 1 end
end
os.exit(count % 256)
